### DISCUSSION TITLE
How to test DivestOS

### DISCUSSION THREAD

**Discussion by edern-appere** [Divested-Mobile/DivestOS-Build#152] [Q&A]
Hi there !

Is there a way to test DivestOS without having to actualy install it on a phone ? Like a VM that I would run on my computer and where I would be able to install apps and see if they do work within DivestOS ?
I've installed DivestOS on my phone a few weeks ago but the entire process of backing up / installing every app and figuring out which one works or not takes a lot of time (and is a little bit risky considering I have only one phone which is my daily driver).

If you have any idea, it would be great !

**Comment by SkewedZeppelin** [comment]
Not that I'm aware.

**Comment by ghost** [comment]
The main apps that wouldn't work are the ones that require google play services and some banking apps right?

**Reply by edern-appere** [reply]
Exactly ! The last time I tried DivestOS, my banking app would not start... Now my bank made a new one, I would have like to test if it works now. That was the only thing preventing me from switching to DivestOS :-/

**Reply by ghost** [reply]
Tbh it probably still won't work. I think there's some safetynet checks that require the os to be certified by google. https://grapheneos.org/usage#banking-apps

I'm guessing youre on stock os at the moment

**Reply by edern-appere** [reply]
Actually I'm using LineageOs+microg (but without having any Google account set up in microg) and it works. I think it might have something to do with Google's SafetyNet ?

@SkewedZeppelin : I heard about ih8sn, which helps fool safetynet. Is it something that is supported (or at least tolerated) in DivestOS or not ?
